File: 1-two-sum/two-sum.java
class Solution {
    public static int[] twoSum(int[] nums, int target) {
        int arr[]=new int[2];
        for(int i=0;i<nums.length;i++){
            boolean x=false;
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    arr[0]=i;
                    arr[1]=j;
                    x=true;
                    break;
                }
            }
            if(x)
                break;
        }
        return arr;
    }
}

File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector <int>v1;
        map<int,int>m;
        for(int i=0;i<nums.size();i++){
            int first=nums[i];
            int second=target-first;
            if(m.find(second)!=m.end()){
                v1.push_back(i) ;
                v1.push_back(m[second]);
                break;
            }
            else{
                m[first]=i;
            }
        }
        return v1;
    }
};